Library/Homebrew/rubocops/no_fileutils_rmrf.rb

```diff
@@ -0,0 +1,58 @@
+# typed: true
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Homebrew
+      # This cop checks for the use of `FileUtils.rm_f`, `FileUtils.rm_rf`, or `{FileUtils,instance}.rmtree`
+      # and recommends the safer versions.
+      class NoFileutilsRmrf < Base
+        extend AutoCorrector
+
+        MSG = "Use `rm` or `rm_r` instead of `rm_rf`, `rm_f`, or `rmtree`."
+
+        def_node_matcher :fileutils_rm_r_f_tree?, <<~PATTERN
+          (send (const {nil? cbase} :FileUtils) {:rm_rf :rm_f :rmtree} ...)
+        PATTERN
+
+        def_node_matcher :instance_rmtree?, <<~PATTERN
+          (send (lvar ...) :rmtree ...)
```

Comment:
I forget what `lvar` covers. Does it handle these cases?

```
Pathname("/a/b/c").rmtree
buildpath.rmtree # where buildpath is a method that returns a Pathname
```

Reply:
It does not. How many of those do we have vs the effort to detect and fix them? Or can we ship this as-is and fix that later?

Reply:
I'd assume it's more common than `lvar`, so up to you.

For detection: it would likely be a case of accepting any receiver - can technically have false positives but it is unlikely we're using `rmtree` for anything not Pathname and RuboCop themselves do similar things all the time.

Reply:
Thanks; done!